### DISCUSSION TITLE
Add channel/system messages support

### DISCUSSION THREAD

**Discussion by imcouri** [flyerhq/flutter_chat_ui#98] [Ideas & Requests]
Hello,
I was wondering if there is a way to add channel messages that notify users about changes/updates in the room using this package.
Thank you

**Comment by demchenkoalex** [comment]
Hi @imcouri you are talking about `SystemMessage` we are plan to introduce. At this point of time you can try to use `CustomMessage` to do whatever you want.

**Comment by demchenkoalex** [comment]
more context: https://github.com/flyerhq/flutter_chat_ui/issues/573

**Comment by demchenkoalex** [comment]
more context: https://github.com/flyerhq/flutter_chat_ui/issues/620
